[2025-01-17T13:50:23.164+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T13:50:23.276+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: detect_data_drift_notify_retrain.prepare_email_no_drift_task manual__2025-01-17T13:39:22.370208+00:00 [queued]>
[2025-01-17T13:50:23.322+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: detect_data_drift_notify_retrain.prepare_email_no_drift_task manual__2025-01-17T13:39:22.370208+00:00 [queued]>
[2025-01-17T13:50:23.326+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-17T13:50:23.407+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): prepare_email_no_drift_task> on 2025-01-17 13:39:22.370208+00:00
[2025-01-17T13:50:23.444+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=467) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-17T13:50:23.463+0000] {standard_task_runner.py:72} INFO - Started process 472 to run task
[2025-01-17T13:50:23.448+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'detect_data_drift_notify_retrain', 'prepare_email_no_drift_task', 'manual__2025-01-17T13:39:22.370208+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/monitoring-dag.py', '--cfg-path', '/tmp/tmpcyhjts0m']
[2025-01-17T13:50:23.466+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask prepare_email_no_drift_task
[2025-01-17T13:50:23.726+0000] {task_command.py:467} INFO - Running <TaskInstance: detect_data_drift_notify_retrain.prepare_email_no_drift_task manual__2025-01-17T13:39:22.370208+00:00 [running]> on host f68435ca3fd9
[2025-01-17T13:50:24.113+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='RL' AIRFLOW_CTX_DAG_ID='detect_data_drift_notify_retrain' AIRFLOW_CTX_TASK_ID='prepare_email_no_drift_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T13:39:22.370208+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T13:39:22.370208+00:00'
[2025-01-17T13:50:24.115+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T13:50:24.351+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T13:50:24.412+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T13:50:24.418+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=detect_data_drift_notify_retrain, task_id=prepare_email_no_drift_task, run_id=manual__2025-01-17T13:39:22.370208+00:00, execution_date=20250117T133922, start_date=20250117T135023, end_date=20250117T135024
[2025-01-17T13:50:24.639+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T13:50:24.801+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-17T13:50:24.811+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
